src/software/ai/hl/stp/evaluation/calc_best_shot.h

```diff
@@ -19,15 +20,16 @@ namespace Evaluation
      * @param obstacles The locations of any obstacles on the field that may obstruct the
      * shot. These are treated as circular obstacles, and are usually used to represent
      * robots on the field.
-     * @param radius The radius of the given obstacles
+     * @param radius The radius of the given obstacles>
```

Comment:
nit: somehow there is an extra angle bracket here

Reply:
Fixed